### PR TITLE
Change several headings to Sentence case

### PR DESCRIPTION
**Pull request by therealseanwallace**
## Because
Update various headings in Javascript course to align with style guide requirements to use Sentence case.

## This PR
Change headings in JavaScript in the Real World && Asynchronous JavaScript and APIs to Sentence case.

## Issue
Related to #25861

## Pull Request Requirements
-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: javascript/javascript_in_the_real_world/linting.md
Comment (ManonLef) [inline javascript/javascript_in_the_real_world/linting.md:1]: I noticed one quirk:
line 45
### Using ESlint and Prettier

This should be:

### Using ESLint and Prettier 

Would you mind changing that as well? 

Thanks for doing this! All other files look perfect!

**Review by ManonLef — CHANGES_REQUESTED** [review]
Besides the `ESLint` issue, this looks great. If that can be fixed, this one is good to merge

**Comment by therealseanwallace** [comment]
Well spotted @ManonLef, that quirk escaped my notice! I just added a new commit fixing it to this pull request.

**Review by ManonLef — APPROVED** [review]

**Comment by ManonLef** [comment]
Thank you so much @therealseanwallace !

**Comment by therealseanwallace** [comment]
Thank you for the opportunity to contribute 😊

**Pull request merged**